library/clefos

```diff
@@ -1,7 +1,7 @@
 Maintainers: The ClefOS Project <neale@sinenomine.net> (@nealef)
 GitRepo: https://github.com/nealef/clefos.git
 
-Tags: 7, 7.6.1810, latest
+Tags: 7, 7.7.1908, latest
 GitFetch: refs/heads/7.4.1708
 GitCommit: 58fdf15e6116ee402029f005b2561b3f23bd6a6d
```

Comment:
I think you probably intended to update this value? :smile:

(Although I'm not seeing https://github.com/nealef/clefos/tree/7.4.1708 updated, which is where the previous tarball was published, so maybe it's not pushed yet?)

Reply:
I have switched branches, applied changes, committed, and pushed.  Do I need to generate another PR?

Reply:
Nope, you just need to update this one!

Reply:
https://github.com/nealef/clefos/tree/7.4.1708 looks good now, you just need to update this PR's `GitCommit:` value to that new commit:

```suggestion
GitCommit: 1aa7d3771b2ced8b8b5cdc9a1a1752d93c56a60e
```

:+1: